src/main/java/com/google/sps/servlets/Login.java

```diff
@@ -0,0 +1,52 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import com.google.sps.data.Authentication;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+
+@WebServlet("/login")
+public class Login extends HttpServlet {
+
+    private static final String REDIRECT_LOGIN = "/";
+    private static final String REDIRECT_LOGOUT = "/";
```

Comment:
Seems weird, why are these just backslashes?

Reply:
I want it to just redirect to the default page which is at /, should I rename the variable to like REDIRECT_AFTER_LOGIN?

Reply:
I see, then maybe add a comment explaining that. // Redirects to default page

Reply:
Sounds great, just added that in!